imctools/converters/ome2analysis.py

```diff
@@ -175,26 +175,26 @@ def omefolder_to_analysisfolder(
 
     tic = timeit.default_timer()
 
-    omefile_2_analysisfolder(
-        Path(
-            "/home/anton/Downloads/imc_folder/20170905_Fluidigmworkshopfinal_SEAJa/20170905_Fluidigmworkshopfinal_SEAJa_s0_a0_ac.ome.tiff"
-        ),
-        Path("/home/anton/Downloads/analysis_folder"),
-        "test",
-        panel_csv_file=Path("/home/anton/Downloads/example_panel.csv"),
-        metalcolumn="Metal Tag",
-        usedcolumn="ilastik",
-    )
-
-    # omefolder_to_analysisfolder(
-    #     "/home/anton/Downloads/imc_folder",
-    #     "/home/anton/Downloads/analysis_folder",
-    #     "/home/anton/Downloads/example_panel.csv",
-    #     [
-    #         ("ilastik", "_ilastik", True),
-    #         ("full", "_full", False)
-    #     ],
+    # omefile_2_analysisfolder(
+    #     Path(
+    #         "/home/anton/Downloads/imc_folder/20170905_Fluidigmworkshopfinal_SEAJa/20170905_Fluidigmworkshopfinal_SEAJa_s0_a0_ac.ome.tiff"
+    #     ),
+    #     Path("/home/anton/Downloads/analysis_folder"),
+    #     "test",
+    #     panel_csv_file=Path("/home/anton/Downloads/example_panel.csv"),
     #     metalcolumn="Metal Tag",
+    #     usedcolumn="ilastik",
     # )
 
+    omefolder_to_analysisfolder(
```

Comment:
Hard-coded file paths to Anton's user directory

Reply:
Yes, I am aware of it. These are used for debugging only, as it was with v1.